File: Company/src/main/webapp/service/tPodaciSubjektService.js
/**
 * Created by JELENA on 20.6.2017.
 */

app.service('tPodaciSubjektService', function ($http) {
    return {
        read: function (onSuccess, onError) {
            $http.get('/api/tPodaciSubjekti').then(onSuccess, onError);
        },
        readPoslovniPartneri: function (id, onSuccess, onError) {
        $http.get('api/tPodaciSubjekti/poslovniPartneri/' + id).then(onSuccess, onError);
        }
    }
});

File: Company/src/main/java/com/ftn/service/TPodaciSubjektService.java
package com.ftn.service;

import com.ftn.model.dto.TPodaciSubjektDTO;

import java.util.List;
import java.util.Set;

/**
 * Created by JELENA on 19.6.2017.
 */
public interface TPodaciSubjektService {

    List<TPodaciSubjektDTO> read();

    List<TPodaciSubjektDTO> readPoslovniPartneri(Long id);

    TPodaciSubjektDTO create(TPodaciSubjektDTO tPodaciSubjektDTO);
}
